src/test/java/seedu/address/storage/ImportManagerTest.java

```diff
@@ -45,6 +45,64 @@ public void importCsvFileAndConvertToJsonFile_validCsv_convertsToJson() throws I
         assertEquals(expectedJson, actualJson);
     }
 
+    @Test
```

Comment:
Could clean up test strings by using a similar format to CommandTestUtil, where the test strings are created with class variables (i.e String testCsvLine = VALID_ID+VALID_NAME....) so future tests are eaiser to generate, and it is easier to see what is invalid at a glance